### PR TITLE
Use web-animations in dancing-dots

### PR DESCRIPTION
**Pull request by dstoc**
Fixes https://github.com/PolymerLabs/arcs/issues/539

### CODE REVIEW

**Comment by dstoc** [comment]
I also changed the animation slightly to remove the delay between motions, as this feels a bit more natural to me. It's not hard to add that back if it's desirable.

File: dev/components/dancing-dots.html [hunk 1]
```diff
@@ -77,21 +67,31 @@
...
     }
     _start() {
-      if (!this._interval) {
-        this.disabled = false;
-        this.setAttribute('animate', '');
-        let animate = () => {
-          this.setAttribute('up', '');
-          setTimeout(() => this.removeAttribute('up'), 280);
+      this.querySelectorAll('bullet').forEach((bullet, i) => {
+        let keyframes = {transform: ['translateY(8px)', 'translateY(-8px)']};
```
Comment (shaper) [inline dev/components/dancing-dots.html]: this and next can be `const`, some in next edited code block as well.

File: dev/components/dancing-dots.html [hunk 2]
```diff
@@ -77,21 +67,31 @@
...
-      }
+        bullet._animation = bullet.animate(keyframes, timing);
+      });
     }
     _stop() {
-      this.removeAttribute('animate');
-      if (this._interval) {
-        clearInterval(this._interval);
-        this._interval = null;
+      for (let bullet of this.querySelectorAll('bullet')) {
```
Comment (shaper) [inline dev/components/dancing-dots.html]: Could use a helper method like `_bulletElements` that returns `this.querySelectorAll('bullet')` here and above to share the string constant and querySelectorAll call.

File: dev/components/dancing-dots.html [hunk 3]
```diff
@@ -77,21 +67,31 @@
...
+        // Ideally we would cause the animation to reach the half way point
+        // between iterations, but that's complicated without the setTiming
+        // API.
+        let transform = getComputedStyle(bullet).transform;
+        let animation = bullet._animation;
+        let stopAnimation = bullet.animate(
+            {transform: [transform, 'translateY(0px)']},
+            {duration: 166.66, easing: 'ease'});
```
Comment (shaper) [inline dev/components/dancing-dots.html]: Maybe a comment on where 166.66 comes from?

**Review by shaper — APPROVED** [review]

**Pull request merged**